CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+## 0.27.0 (2022-10-20)
```

Comment:
Why is it a minor version bump rather than a patch release, if all it includes is updated dependencies? If the upstream SDK changes introduce new features, it would be good to refer to them here.

Reply:
Yep, good points. I'll add some more details about the API changes.